### ISSUE TITLE
[A11y] - Target does not meet minimum size or spacing

### ISSUE THREAD

**Issue by VaniChinnam** [dequelabs/cauldron#1396]
### Current Behaviour 
`Target does not meet minimum 24 x 24 px size or spacing. `<br/><hr/>
### Expected Behaviour 
`RULE :  Targets MUST have an area of at least 24 by 24 CSS pixels or are spaced apart so that a 24 pixel diameter circle centered on the target does not touch another target nor a 24 pixel diameter circle placed on the center of any other adjacent targets that are less than 24 by 24 pixels. Exceptions exist. HOW TO FIX:  Fix this issue by doing at least ONE of the following: 1. Ensure that the control has a target size of at least 24 x 24 CSS px. 2. Ensure that a 24 pixel diameter circle centered on the target does not touch another target nor a 24 pixel diameter circle placed on the center of any other adjacent targets that are less than 24 by 24 pixels `<br/><hr/>
### Impact 
 <p>

- [ ] Blocker
- [x] Serious
- [ ] Critical
- [ ] Moderate
- [ ] Minor

### Issue Details

<details><summary>View Issue 1658554 </summary> 
 <p>

_**Auditor ID**_ : [1658554](https://axeauditor.dequecloud.com/issue-global-search?searchKeyword=1658554&searchOn=id) 

_**description**_ :  List box (716 *18px) - Target doesn't meet the minimum size

Element location: Below the following headings: 
-Listbox Options 
-Disabled Options 
-Grouped Options 
-Keyboard Navigation 
-Controlled -Uncontrolled  

_**Page Name**_ : Listbox 

_**source**_ : `<li id="listbox-option1" class="" role="option" aria-selected="false">One</li>` 

_**Screenshots**_ : [Target does not meet minimum size or spacing](https://axeauditor.dequecloud.com/api/v1/file/139ca586-bf88-11ee-ae15-a71978afb237
Target does not meet minimum size or spacing: https://axeauditor.dequecloud.com/api/v1/file/139cbce2-bf88-11ee-b412-b34510b49716) 

</p>
</details><details><summary>View Issue 1688398 </summary> 
 <p>

_**Auditor ID**_ : [1688398](https://axeauditor.dequecloud.com/issue-global-search?searchKeyword=1688398&searchOn=id) 

_**description**_ :  copy code example to clipboard (23*23px) 
Element location: In the pages - where there is a scrollbar for code block.
- Radio Card Group page -> Below the "Examples" 
- Checkbox page -> Below the "Checkbox with Description List" 
- Radio group page -> Below the "Examples" 
etc.,.

_**Page Name**_ : Project Wide 

_**source**_ : `<button class="IconButton IconButton--secondary" tabindex="0" type="button" data-copy-example="true" aria-label="copy code example to clipboard, 1 of 1" aria-labelledby="tooltip57"><span aria-hidden="true" class="Icon Icon--copy"><svg overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"></svg></span></button>` 

_**Screenshots**_ : [image](https://axeauditor.dequecloud.com/api/v1/file/ce67fbac-dd3f-11ee-bbb4-17334141300e
image: https://axeauditor.dequecloud.com/api/v1/file/ce67fbde-dd3f-11ee-a7b3-ff447b5ef62c
image: https://axeauditor.dequecloud.com/api/v1/file/ce6aaf28-dd3f-11ee-989c-533dfdb6a8fc
image: https://axeauditor.dequecloud.com/api/v1/file/ce6ab84c-dd3f-11ee-9dcf-2f4f68f6ae52
image: https://axeauditor.dequecloud.com/api/v1/file/ce6b070c-dd3f-11ee-94c9-97a101fa8234
image: https://axeauditor.dequecloud.com/api/v1/file/ce6ba518-dd3f-11ee-87ac-8f69604845b7
image: https://axeauditor.dequecloud.com/api/v1/file/ec56e52a-dcb2-11ee-a346-134b67ebd128) 

</p>
</details>
